packages/shared/prisma/schema.prisma

```diff
@@ -635,6 +636,27 @@ model Events {
     @@map("events")
 }
 
+model Comment {
+    id           String            @id @default(cuid())
+    projectId    String            @map("project_id")
+    project      Project           @relation(fields: [projectId], references: [id], onDelete: Cascade)
+    objectType   CommentObjectType @map("object_type")
+    objectId     String            @map("object_id")
+    timestamp    DateTime          @default(now())
+    content      String
+    authorUserId String?           @map("author_user_id")
+
+    @@index([projectId, objectType, objectId])
+    @@map("comments")
+}
```

Comment:
I considered the cost/benefit of adding a uniqueness constraint on DB level and deemed it v. slightly net negative here as it requires the database to check for existing values before inserting/updating a record and does not provide much value given these are simple comments -- but ultimately I don't have strong feelings here, wdyt?

Reply:
on what do you want to check uniqueness here? on id will happen automatically, is there another uniqueness that you'd like to enforce?